src/spaceone/identity/error/error_user.py

```diff
@@ -31,3 +31,23 @@ class ERROR_INCORRECT_USER_ID_FORMAT(ERROR_INVALID_ARGUMENT):
 
 class ERROR_NOT_ALLOWED_ACTIONS(ERROR_INVALID_ARGUMENT):
     _message = 'External or API user are supported for actions. (action = {action})'
+
+
+class ERROR_INVALID_VERIFY_CODE(ERROR_INVALID_ARGUMENT):
+    _message = 'Invalid verify token. (verify_code = {verify_code})'
+
+
+class ERROR_UNABLE_TO_RESET_PASSWORD(ERROR_INVALID_ARGUMENT):
+    _message = 'Unable to reset password. (user_id = {user_id})'
+
+
+class ERROR_PASSWORD_NOT_CHANGED(ERROR_UNABLE_TO_RESET_PASSWORD):
```

Comment:
change the message to
`The requested password is the same as the previous password.`